docs/reference/commandline/network_create.md

```diff
@@ -120,6 +121,11 @@ docker network create -d overlay
 ```
 Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
 
+### Network internal mode
+
+By default, when you connect a container to an `overlay` network, docker also connects a bridge network to it to provide external connectivity.
+If you want to create an externally isolated `overlay` network, you can specify the `--internal` option.
```

Comment:
@thaJeztah 

> question: does this flag only apply to overlay networks?

No. This flag is applicable to all the drivers (and yes also to custom drivers).

> does it error out if I apply it to a bridge network?

Currently (at the time of writing this comment), the bridge driver will error out. But it will be fixed shortly (before 1.10 code-freeze)

> What happens if I apply it to a custom driver?

If the driver makes use of the gateway service from libnetwork, then it will be handled automatically by the libnetwork core. But if the custom driver provides its own service then it must handle the internal flag to provide that isolation. It is similar to any other top-level network configuration.